splice_machine/src/main/java/com/splicemachine/derby/jdbc/SpliceTransactionResourceImpl.java

```diff
@@ -51,14 +51,18 @@
     protected LanguageConnectionContext lcc;
     protected String ipAddress;
 
-    public SpliceTransactionResourceImpl() throws SQLException{
-        this(CONNECTION_STRING, new Properties());
+    public SpliceTransactionResourceImpl() throws SQLException{ // XXX Remove this one and properly pass the right DB
```

Comment:
Should be removed?:)

Reply:
This will be addressed in the following jiras